go.mod

```diff
@@ -8,11 +8,11 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-openapi/spec v0.20.3
 	github.com/gogo/googleapis v1.4.1 // indirect
-	github.com/gogo/protobuf v1.3.2
+	github.com/gogo/protobuf v1.3.2 // indirect
```

Comment:
hmm, something still requires gogo then?

Reply:
Some of out plugins still use gogo in the backend, but the generated code shouldn't (unless it calls code from the plugin repo and causes an additional indirect import)